### PR TITLE
Split jwtsso fat

### PR DESCRIPTION
**Pull request by c00crane**
Split the jwtsso fat.
This fat runs the same tests multiple times - each run specifies a different setup of mpJwt.
It runs without mpJwt (and that runs with and without EE9)
It runs with mpJwt 1.1, mpJwt 1.2 and mpJwt 2.0.  This just got too large and time consuming.
com.ibm.ws.security.jwtsso_fat			
has now become:
com.ibm.ws.security.jwtsso_fat.commonTest - contains the tests
com.ibm.ws.security.jwtsso_fat.mpJwt-1.1 - runs with mpJwt 1.1
com.ibm.ws.security.jwtsso_fat.mpJwt-1.2 - runs with mpJwt 1.2
com.ibm.ws.security.jwtsso_fat.mpJwt-2.0 - runs with mpJwt 2.0 (which include EE9)
com.ibm.ws.security.jwtsso_fat.noMpJwt - has no mpJwt in the config and runs with/without EE9

### CODE REVIEW

**Comment by c00crane** [comment]
#build

**Comment by c00crane** [comment]
#run-libby-bot

**Comment by c00crane** [comment]
Personal build with updates for noMpJwt_ee9
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_pf7UgEGQEeyda8UYcX_yfQ

**Comment by c00crane** [comment]
requested rebuild of the wrong build, so, that explains why the same failures occurred - the build didn't have the last pull

**Comment by c00crane** [comment]
#run-libby-bot

**Comment by c00crane** [comment]
#build

File: dev/com.ibm.ws.security.jwtsso_fat.commonTest/fat/src/com/ibm/ws/security/jwtsso/fat/Something.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                FeatureOnlyTest.class,
+//                ConfigAttributeTests.class,
+//                CookieProcessingTests.class,
+//                ReplayCookieTests.class,
+//                CookieExpirationTests.class,
+//                BuilderTests.class,
+//                SigAlgTests.class,
+////                EncryptionTests.class
+})
+public class Something {
```
Comment (ayoho) [inline dev/com.ibm.ws.security.jwtsso_fat.commonTest/fat/src/com/ibm/ws/security/jwtsso/fat/Something.java]: What's the purpose of this class?

**Review by ayoho — APPROVED** [review]
Note that a few file permissions got changed from 755 to 644.

File: dev/com.ibm.ws.security.jwtsso_fat.commonTest/fat/src/com/ibm/ws/security/jwtsso/fat/Something.java (hunk 1 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.jwtsso_fat.commonTest/fat/src/com/ibm/ws/security/jwtsso/fat/Something.java]: It was my backup of the fatsuite that I was planning on removing.
Removing in a new pull

Comment (c00crane) [reply dev/com.ibm.ws.security.jwtsso_fat.commonTest/fat/src/com/ibm/ws/security/jwtsso/fat/Something.java]: The file permission change was something Windows did - I'm trying to reset those.

**Comment by c00crane** [comment]
#build

**Comment by c00crane** [comment]
#run-libby-bot

**Comment by c00crane** [comment]
#build

**Comment by c00crane** [comment]
#build

**Comment by wraschke** [comment]
@c00crane we had to revert this because it is causing a hard break in LAOS continuous. Please see https://ibm-cloud.slack.com/archives/C30NGUQ9E/p1637759483421000 for details.

**Pull request merged**